sharding/node/backend.go

```diff
@@ -93,25 +93,26 @@ func New(ctx *cli.Context) (*ShardEthereum, error) {
 }
 
 // Start the ShardEthereum service and kicks off the p2p and actor's main loop.
-func (s *ShardEthereum) Start() {
+func (s *ShardEthereum) Start() error {
 	log.Info("Starting sharding node")
 	for kind, service := range s.services {
 		// Start the next service.
 		if err := service.Start(); err != nil {
 			log.Error(fmt.Sprintf("Could not start service: %v, %v", kind, err))
+			s.Close()
 		}
 	}
+	return nil
 }
 
 // Close handles graceful shutdown of the system.
-func (s *ShardEthereum) Close() error {
+func (s *ShardEthereum) Close() {
 	for kind, service := range s.services {
 		if err := service.Stop(); err != nil {
-			return fmt.Errorf(fmt.Sprintf("Could not stop service: %v", kind))
+			log.Info(fmt.Sprintf("Could not stop the following service: %v, %v", kind, err))
```

Comment:
Info? This needs to be log level Error or Critical, whichever is higher.

Some servers may filter out info level logs.